sentry-types/src/protocol/envelope.rs

```diff
@@ -59,6 +59,64 @@ struct EnvelopeHeaders {
     trace: Option<DynamicSamplingContext>,
 }
 
+impl EnvelopeHeaders {
+    /// Creates empty Envelope headers.
+    pub fn new() -> EnvelopeHeaders {
+        Default::default()
+    }
+
+    /// Returns the Event ID.
+    pub fn event_id(&self) -> Option<&Uuid> {
```

Comment:
As discussed offline, if we don't expect that the getters will be needed, let's not include them in the public API